Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -522,6 +545,8 @@ public void StopServer()
             if (!NetworkServer.active)
                 return;
 
+            authenticator?.OnServerAuthenticated.RemoveListener(OnServerAuthenticated);
```

Comment:
?. doesn't work for unity objects/components. this gives a warning in Rider at least. better to check != null first

Reply:
Works perfect using breakpoints in visual studio:

Before
![image](https://user-images.githubusercontent.com/9826063/65014875-e7d73400-d8ed-11e9-9e26-1e025f557211.png)

After
![image](https://user-images.githubusercontent.com/9826063/65014930-181ed280-d8ee-11e9-8fd1-2079ae397978.png)

Reply:
Doesn't work as in, doesn't return correct values

Reply:
@MrGadget1024 if I remember correctly, this was about after components/GOs were destroyed. the pointer would still be != null, but the component was destroyed and should be considered null. that's why unity overwrites the == check and why we need to use it

Reply:
changed it